raster/src/main/scala/geotrellis/raster/Implicits.scala

```diff
@@ -24,33 +24,33 @@ object Implicits extends Implicits
 
 trait Implicits
     extends costdistance.Implicits
-    with crop.Implicits
-    with density.Implicits
-    with distance.Implicits
-    with equalization.Implicits
-    with hydrology.Implicits
-    with interpolation.Implicits
-    with io.json.Implicits
-    with mapalgebra.focal.Implicits
-    with mapalgebra.focal.hillshade.Implicits
-    with mapalgebra.local.Implicits
-    with mapalgebra.zonal.Implicits
-    with mask.Implicits
-    with matching.Implicits
-    with merge.Implicits
-    with prototype.Implicits
-    with rasterize.Implicits
-    with regiongroup.Implicits
-    with render.Implicits
-    with reproject.Implicits
-    with resample.Implicits
-    with sigmoidal.Implicits
-    with split.Implicits
-    with summary.Implicits
-    with summary.polygonal.Implicits
-    with transform.Implicits
-    with vectorize.Implicits
-    with viewshed.Implicits {
+    with geotrellis.raster.crop.Implicits
```

Comment:
Yea that's what I remember doing, I'd like to review more a bit later, since I definitely played around with it but don't remember to what conclusion I came.

Reply:
I think replacing the _ imports with explicit imports is maybe a better option, though we could do both.